test/functional/rpc_signrawtransaction.py

```diff
@@ -50,6 +51,7 @@ def successful_signing_test(self):
 
     def test_with_lock_outputs(self):
         """Test correct error reporting when trying to sign a locked output"""
```

Comment:
This comment is redudant to the introduced logging message and could be removed? (Same as for `witness_script_test()` below).

Reply:
done